libs/coin-modules/coin-tezos/src/types/signer.ts

```diff
@@ -0,0 +1,56 @@
+export type TezosAddress = {
+  address: string;
+  publicKey: string;
+};
+export type TezosSignature = {
+  signature: string;
+};
+// Type coming from hw-app-tezos
+export const TezosCurves = {
+  ED25519: 0x00,
+  SECP256K1: 0x01,
+  SECP256R1: 0x02,
+};
+export type Curve = (typeof TezosCurves)[keyof typeof TezosCurves];
```

Comment:
```suggestion
type TezosCurves = {
  ED25519: 0x00;
  SECP256K1: 0x01;
  SECP256R1: 0x02;
};

export type Curve = TezosCurves[keyof TezosCurves];
```

TezoCurves is not used outside of module no need to export it